pygpsclient/serialconfig_frame.py

```diff
@@ -53,7 +53,7 @@
 }
 # These ranges can be overridden via keyword arguments:
 # (the default value will be the first in the range)
-BPSRATE_RNG = (9600, 19200, 38400, 57600, 115200, 4800)
+BPSRATE_RNG = (9600, 19200, 38400, 57600, 115200, 4800, 1000000)

```

Comment:
as per previous comment - why the additional non-standard baud rate, and can we keep them in ascending order?